Se 13419 rtm infinite link loop hn (#216)
mulesoft/docs-runtime-manager: modules/ROOT/pages/cluster-create.adoc

= Create Clusters in Runtime Manager (Hybrid)
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]

> To deploy applications to a cluster, you can either add the servers to Runtime Manager first, then create the cluster or you can create the cluster and add servers to it later.

Reviewer: ```suggestion
To deploy applications to a cluster, you can either add the servers to Runtime Manager first and then create the cluster, or you can create the cluster and add servers to it later.
```

For Object Store persistence on all the servers in your cluster, create your cluster manually and use Quorum Management.

== Restrictions

> To add servers to a cluster:

Reviewer: ```suggestion
Before you add servers to a cluster, note the following restrictions:
```

* Servers cannot contain any previously deployed applications.
* Servers cannot belong to another cluster or group.

> * Multicast servers can be in the *Running* or *Disconnected* state. 

Reviewer: No reason to boldface server states; capitalization is sufficient, and we are not overtly quoting the UI.
```suggestion
* Multicast servers can be in either the Running or the Disconnected state. 
```

> * Unicast servers must be in the *Running* state. 

Reviewer: ```suggestion
* Unicast servers must be in the Running state. 
```

* All servers in a cluster must be running the same Mule runtime engine version and Runtime Manager agent version.

== Create a Cluster

To create a cluster:

. From Anypoint Platform, select *Runtime Manager*.
. Select *Servers* in the left menu.
. Click the *Create Cluster* button:
+
.The arrow shows the *Create Cluster* button on the *Servers* page.
image::rtm-cluster-create.png[Create Cluster button]

. In the *Create Cluster* page, enter the name for the cluster.
+

> Cluster names can contain between 3-40 alphanumeric characters (a-z, A-Z, 0-9) and hyphens (-). They cannot start or end with a hyphen and cannot contain spaces or other characters.

Reviewer: ```suggestion
Cluster names can contain from 3 through 40 alphanumeric characters (a-z, A-Z, 0-9) and hyphens (-). They cannot start or end with a hyphen and cannot contain spaces or other characters.
```

Author: disagree
